src/Compiler/CompilerOptions.cpp

```diff
@@ -337,13 +337,16 @@ static llvm::cl::opt<bool, true> storeConstantsToFileOpt(
     "store-constants-to-file",
     llvm::cl::desc(
         "Constants will be stored on a binary file instead of be embedded "
-        "into the model.so. The binary file is in the same folder as the "
-        "model.so and has the same name as the model with the extension of "
-        ".constants.bin. For inference, model.constants.bin must be at the "
-        "same folder as the inference program. If model.constants.bin is at "
-        "another folder, use the environment variable OM_CONSTANT_PATH to set "
-        "the constant folder. Windows will be supported soon."),
-    llvm::cl::location(storeConstantsToFile), llvm::cl::init(false),
+        "into the model.so when the sizes of constants satisfy thresholds "
+        "defined by options constants-to-file-single-threshold and "
+        "constants-to-file-total-threshold. The binary file is in the same "
```

Comment:
Nit: I would add "When using the model, "... to specify when that co-location and/or env variable must be defined.

Reply:
I added this `When using this option, two other options constants-to-file-single-threshold and constants-to-file-total-threshold can be used to finetune the amount of constants stored on the file.`. Hope it's fine. Thanks!